src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests4.cs

```diff
@@ -1718,15 +1718,15 @@ static int M1(bool? b1, bool? b2)
             (true, false) => 3,
             (true, true) => 4,
             _ => 5,
-            (null, true) => 6
+            (null, true) => 6,
             };
     }
 }
 ";
             var compilation = CreatePatternCompilation(source);
             compilation.VerifyDiagnostics(
                 // (13,13): error CS8510: The pattern has already been handled by a previous arm of the switch expression.
-                //             (null, true) => 6
+                //             (null, true) => 6,
```

Comment:
Consider adding a test with two trailing commas. (same feedback also applies to property pattern)